base/compiler/abstractinterpretation.jl

```diff
@@ -1380,6 +1380,12 @@ function abstract_eval_special_value(interp::AbstractInterpreter, @nospecialize(
         return (vtypes[slot_id(e)]::VarState).typ
     elseif isa(e, GlobalRef)
         return abstract_eval_global(e.mod, e.name)
+    elseif isa(e, PhiNode)
```

Comment:
I think it would be better to move this to `abstract_eval_statement`, since it can't be nested in anything.

Reply:
ok